EncoreTickets.SDK/Utilities/HashHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace EncoreTickets.SDK.Utilities
+{
+    public static class HashHelper
+    {
+        public static string CreateMd5Hash(string sourceData)
+        {
+            var md5 = MD5.Create();
+            var inputBytes = sourceData == null ? new byte[0] : Encoding.UTF8.GetBytes(sourceData);
```

Comment:
Not really sure that this is how this should work. For me sourceData should always be defined and non-null value. Maybe it makes sense to add early return and return null if null was passed in parameters.